websites/T/Tiermaker/presence.ts

```diff
@@ -0,0 +1,92 @@
+let episode: string,
+    creator: string,
+    c_title: string,
+    c_category: string,
+    c_stateNum: number,
+    c_currentState: string;
+
+const presence = new Presence({
+    clientId: "919901175640358983"
+  }),
+
+  startStamp = Math.floor(Date.now() / 1000),
+  urlpath = document.location.pathname.split("/"),
+
+  getElement = (query: string): string | undefined => {
+    return document.querySelector(query)?.textContent;
+  },
+
+  getInformation = (query: string): string | undefined => {
+    episode = document.querySelector(query)?.textContent;
+    return episode
+  };
+
+  
+function c_GetDetails(): string | undefined {

```

Comment:
idk what c stands for but please use camel casing. Also this returns void currently, you should've gotten a TS error
```suggestion
function c_GetDetails(): void {
```